### PR TITLE
Stop accept(..) & reject(..) if request not open

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by dominikkowalski** [comment]
🆗

**Comment by krzysztofzygmunt** [comment]
:ok:

File: powerdns/models/requests.py [hunk 1]
```diff
@@ -138,8 +151,12 @@ def accept(self):
         self.save()
         return object_
 
+    @transaction.atomic
     def reject(self):
         """Reject the request"""
+        if self.state != RequestStates.OPEN:
+            log.warning('Request (id:{}) already accepted'.format(self.id))
```
Comment (mkurek) [inline powerdns/models/requests.py]: accepted -> rejected

Comment (ghost) [inline powerdns/models/requests.py]: change it, here and others

Comment (mkurek) [inline powerdns/models/requests.py]: accepted is not perfect here. either use rejected or sth more generic

**Comment by mkurek** [comment]
:ok:

**Pull request merged**